contrib/functions/go/delete-annotations/main.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
```

Comment:
Suggest using the new SDK. Here's an example https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/769

Reply:
Acked